test/basic/test_lightgbm.jl

```diff
@@ -63,11 +63,14 @@ end
         cp(settings["ref_lib_lightgbm_path"], settings["custom_fixture_path"]) # fake file copied from lightgbm
 
         # Act
-        output = LightGBM.find_library(settings["sample_lib"], [src_dir])
+        push!(Libdl.DL_LOAD_PATH, src_dir)
```

Comment:
Might be better to move these `push!` and `deleteat!` lines into the `setup_env` and `teardown` functions, as this means any additional tests won't need to remember to add these lines.